source/main.cc

```diff
@@ -339,7 +353,20 @@ parse_parameters (const std::string &input_as_string,
   // other processors will be ok as well
   if (dealii::Utilities::MPI::this_mpi_process (MPI_COMM_WORLD) != 0)
     {
+#if DEAL_II_VERSION_GTE(8,5,0)
+      try
+        {
+          prm.parse_input_from_string(input_as_string.c_str());
+        }
+      catch (const dealii::ExceptionBase &e)
+        {
+          success = false;
+          e.print_info(std::cerr);
+          std::cerr << std::endl;
```

Comment:
This is a bit hokey -- we had previously found that we can parse the input just fine from processor 0, so once we get here on all of the other processors, there should be no reason that we can't parse the same input just fine here as well.

I think I would just remove the whole try-catch block, along with setting `success` in line 368 and the assertion in line 370 (new lines numbers). If one of the other processors finds that they can't parse the output, it's probably for a more serious reason -- it can't be because the input file is invalid. So in that case let the exception happen and propagate all the way to the top.